### DISCUSSION TITLE
The problem with restoring blocks after explosions (any)

### DISCUSSION THREAD

**Discussion by Aizofick1** [TownyAdvanced/Towny#6095] [Q&A] [answered]
**After disabling the recovery of blocks in the config after explosions and mobs and TNT blocks are still restored**

**Comment by LlmDl** [comment, marked as answer]
You disabled it in the config which is clearly marked as only being defaults for new worlds. Edit the world files in the towny\data\worlds\ folder.

**Reply by Aizofick1** [reply]
Thanks

**Reply by Aizofick1** [reply]
Please add the ability to reload the plugin with the command /town reload

**Reply by LlmDl** [reply]
the command `/ta reload` already exists.

**Reply by Aizofick1** [reply]
Thanks
